app/_components/utility/pageHeading.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Flex, Heading } from '@chakra-ui/react';
+import GetColor from '@/app/_hooks/colorSelector';
+
+const PageHeading = ({
+  heading,
+  subheading,
+}: {
+  heading: string;
+  subheading: string;
+}) => (
+  <Flex direction="column" gap={2}>
+    <Box alignSelf="baseline">
+      <Heading
+        as="h1"
+        size={{ base: 'md', md: 'lg', lg: 'lg' }}
+        marginX={{ base: 'auto', md: '0', lg: '0' }}
+        color={GetColor('accent-text')}
+      >
+        {heading}
+      </Heading>
+    </Box>
+    <Box alignSelf="baseline">
+      <Heading
+        as="h2"
+        size={{ base: 'xs', md: 'sm', lg: 'sm' }}
+        marginX={{ base: 'auto', md: '0', lg: '0' }}
+        textAlign={{ base: 'center', sm: 'left', md: 'left' }}
+      >
+        {subheading}
+      </Heading>
+    </Box>
```

Comment:
@chrismou would you prefer this subheading to be optional? I noticed you don't always use it.

Reply:
My pick would be to keep it optional. But confirm with @chrismou

Reply:
I could shoehorn subheadings on to all pages, but I don't think they all need them so would probably say optional?